packages/input-date/src/LionInputDate.js

```diff
@@ -24,15 +28,19 @@ export class LionInputDate extends LocalizeMixin(LionInput) {
 
   constructor() {
     super();
-    this.parser = (value, options) => (value === '' ? undefined : parseDate(value, options));
+    /**
+     * @param {string} value
+     */
+    this.parser = value => (value === '' ? undefined : parseDate(value));
```

Comment:
we no longer parse the options?

Reply:
parseDate does not accept any 🙈  it uses localize.locale. There's no other parse options like there is with numbers.

Reply:
hmm ok, but `parseDate` should accept options like a different locale... 😅

but yeah that is for another pull request